styleGuidelines.md

```diff
@@ -373,19 +378,32 @@ struct B {
 }
 ```
 
-## 7. Command block ##
+## 7. Tasks ##
+
+### The command section ###
 1. Each option in a bash command should be on a new line. Ending previous lines
    with a backslash (`\`). Some grouping of options on a single line may be
    acceptable. For example, various java memory settings may be set on the same
    line, as long as the line does not exceed the line length limit as described
    in section 4.
-2. All bash commands should start with `set -eo pipefail`.
-3. All commands should have a `~{preCommand}` following point 2 and before the
-   actual command. Preferably *directly* before the actual command. The
-   preCommand should be settable as an inputs.
+2. All bash commands should start with `set -e -o pipefail` if more than one
+   bash command gets executed in the task.
+3. Commands should have a `~{preCommand}` following point 2 and before the
+   actual command, if no docker container is specified in the runtime section.
```

Comment:
I would voice this more strongly. A docker container should be provided with each task.
If due to legacy or other reasons this is not yet possible a ~{preCommand} etc. But that's just me.